### ISSUE TITLE
Can't build WASM with newer Rust versions

### ISSUE THREAD

**Issue by gbrail** [bazelbuild/rules_rust#403]
If I set rust_repositories in my WORKSPACE to have a "version" flag that is fairly recent -- for instance:

rust_repositories(edition = "2018", version = "1.46.0")

and then I try to build a WASM module using the build-in wasm platform:

bazel build ANYTHING --platforms @io_bazel_rules_rust//rust/platform:wasm

then I get an error when trying to link the final .wasm:

  = note: rust-lld: error while loading shared libraries: libLLVM-10-rust-1.46.0-stable.so: cannot open shared object file: No such file or directory

This is the error on Linux -- on Mac Darwin I get the same type of error but the exact error is of course different.

On both Linux and Darwin, it looks like Bazel is correctly using the rust-lld binary found in the output directory tree under:

external/rust_linux_x86_64/lib/rustlib/x86_64-unknown-linux-gnu/bin

however, this binary now depends on libLLVM, which is a shared library found in (for instance) external/rust_linux_x86_64/lib

Presumably that means setting LD_LIBRARY_PATH on Linux or whatever the Darwin equivalent is.

I'm not sure where in this tree of Bazel rules to figure out how do fix that, but if anyone can point the way I'll give it a try.

**Comment by gbrail** [comment]
To make things more confusing, if I delete the "version" from my rust_repositories rule, then it works (with an older version of Rust) even though in that version, rust-lld also dynamically links libLLVM, but in that case it works. It's only a problem when "version" is set.

**Comment by PiotrSikora** [comment]
This is a duplicate of #387, and it's going to be fixed once #451 is merged.

**Comment by UebelAndre** [comment]
Is this still an issue?

**Comment by UebelAndre** [comment]
Closing this out since the issue appears to be resolved.
